OpenRA.Platforms.Default/OpenGL.cs

```diff
@@ -19,6 +19,7 @@
 
 namespace OpenRA.Platforms.Default
 {
+#pragma warning disable IDE1006 // Naming Styles
```

Comment:
Our other uses of `#pragma warning disable` are indented to match the lines that follow, so can we please add a tab here?